specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -6,22 +6,80 @@ import "../shared/routes.tsp";
 using Azure.Core;
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace DevCenterService;
 
 interface DevBoxes {
-  @doc("Lists available pools")
+  @doc("Lists available pools.")
   listPools is StandardResourceOperations.ResourceList<Pool>;
 
-  @doc("Gets a pool")
+  @doc("Gets a pool.")
   getPool is StandardResourceOperations.ResourceRead<Pool>;
 
-  @doc("Lists available schedules for a pool.")
-  listSchedulesByPool is StandardResourceOperations.ResourceList<Schedule>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Does not fit any standard operation pattern since DevBox has a different path"
+  @doc("Lists Dev Boxes that the caller has access to in the DevCenter.")
+  @route("/devboxes")
+  @get
+  listAllDevBoxes is Azure.Core.Foundations.Operation<{}, DevBoxListResult>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Does not fit any standard operation pattern since DevBox has a different path"
+  @doc("Lists Dev Boxes in the Dev Center for a particular user.")
+  @route("/users/{userId}/devboxes")
+  @get
+  listAllDevBoxesByUser is Azure.Core.Foundations.Operation<
+    {
+      @doc("The AAD object id of the user. If value is 'me', the identity is taken from the authentication context.")
+      @path
+      userId: string;
+    },
+    DevBoxListResult
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Need of reuse ScheduleListResult"
+  // Alternate route to return same list response of Schedules
+  @doc("Lists all schedules within a pool that are configured by your project administrator.")
+  @route("/projects/{projectName}/pools/{poolName}/schedules")
+  @get
+  listSchedulesByPool is Azure.Core.Foundations.Operation<
```

Comment:
We could use Foundations.ResourceList here

Reply:
Did we have to stop using the standard template for this operation because of listSchedulesByProject? I dont think that should have caused a conflict since the routes are different.

Reply:
We stop using to be able to reuse `ScheduleListResult`. When we use stand op for List it creates it's own ListResult . So in the other operations that we can't use stand op we get blocked to re-use the ScheduleListResult because that type has been created for the standop before